xml/System.Resources.Extensions/PreserializedResourceWriter.xml

```diff
@@ -248,10 +305,10 @@
         <Parameter Name="typeName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="typeName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The resource name.</param>
+        <param name="value">The value of the resource in string form understood by the type's <see cref="T:System.ComponentModel.TypeConverter" />.</param>
+        <param name="typeName">An assembly qualified type name of the resource</param>
```

Comment:
```suggestion
        <param name="typeName">The assembly qualified type name of the resource</param>
```

Reply:
I edited your suggestion to include the dot at the end.